features/reporting.yml

```
@@ -1,7 +1,39 @@
 name: Reporting API
-description: The Reporting API makes consistent reports about Content Security Policy violations, Permissions-Policy violations, deprecated feature usage, crashes, and other web platform features available to your server endpoints.
-spec: https://w3c.github.io/reporting/
+description: The `Reporting-Endpoints` HTTP header specifies URLs to receive reports of Content Security Policy (CSP) violations, Permissions-Policy violations, deprecated feature usage, and crashes, while the `ReportingObsever()` API dispatches reports to a callback function.
```

I know we usually try to mention entry points first in the description, but reversing the order of things makes it clearer to me. Up to you which one you prefer:

```suggestion
description: Content Security Policy (CSP) violations, Permissions-Policy violations, deprecated feature usage, and crashes can be sent to server endpoints by using the `Reporting-Endpoints` HTTP header, and to JavavaScript callback functions by using the `ReportingObserver()` API
```

---

I'm realizing that the description doesn't mention browser interventions. If I'm correct, there are 5 types of reports: csp, deprecation, crashes, permissions, and interventions. If we list the first 4, we should add the 5th to the list.

---

I did appreciate your revision, though. It's a good way to frame it. Here's my new take on this, that tries to meld both approaches better:

```suggestion
description: The `Reporting-Endpoints` HTTP header and `ReportingObsever()` API send reports of Content Security Policy (CSP) violations, Permissions-Policy violations, deprecated feature usage, and crashes to a nominated URL or callback function.
```

Or with interventions:

```suggestion
description: The `Reporting-Endpoints` HTTP header and `ReportingObsever()` API send reports of Content Security Policy (CSP) violations, Permissions-Policy violations, deprecated feature usage, browser interventions, and crashes to a nominated URL or callback function.
```